tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,140 @@
+    databaseUser: ironic
+    passwordSelectors:
+      database: IronicDatabasePassword
+      service: IronicPassword
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic
+    standalone: false
+  ironicConductor:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic
+    passwordSelectors:
+      database: IronicDatabasePassword
+      service: IronicPassword
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic
+    standalone: false
+    storageClass: local-storage
+    storageRequest: 10G
+  ironicInspector:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-inspector:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic_inspector
+    passwordSelectors:
+      database: IronicInspectorDatabasePassword
+      service: IronicInspectorPassword
+    preserveJobs: true
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    rabbitMqClusterName: rabbitmq
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic-inspector
+    standalone: false
+  passwordSelectors:
+    database: IronicDatabasePassword
+    service: IronicPassword
+  preserveJobs: true
+  rabbitMqClusterName: rabbitmq
+  rpcTransport: json-rpc
+  secret: osp-secret
+  serviceUser: ironic
+  standalone: false
+status:
+  apiEndpoints:
+    ironic:
+      admin: http://ironic-admin-openstack.apps-crc.testing
+      internal: http://ironic-internal-openstack.apps-crc.testing
+      public: http://ironic-public-openstack.apps-crc.testing
+    ironic-inspector:
+      admin: http://ironic-inspector-admin-openstack.apps-crc.testing
+      internal: http://ironic-inspector-internal-openstack.apps-crc.testing
+      public: http://ironic-inspector-public-openstack.apps-crc.testing
+  conditions:
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: Ready
+  # - message: Bootstrap not started
+  #   reason: Init
+  #   status: Unknown
+  #   type: BootstrapReady
+  - message: DB create completed
+    reason: Ready
+    status: "True"
+    type: DBReady
+  - message: DBsync completed
+    reason: Ready
+    status: "True"
+    type: DBSyncReady
+  # - message: Deployment not started
+  #   reason: Init
+  #   status: Unknown
+  #   type: DeploymentReady
+  # - message: Exposing service not started
+  #   reason: Init
+  #   status: Unknown
+  #   type: ExposeServiceReady
+  - message: Input data complete
+    reason: Ready
+    status: "True"
+    type: InputReady
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: IronicAPIReady
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: IronicConductorReady
+  - message: IronicRabbitMqTransportURL disabled
```

Comment:
Just a thought here, it only makes sense to have this if we're going to always assert json-rpc as the default, which is not the case in a fully integrated configuration.

Reply:
Aligning with other openstack-k8s-operators - in the latest patch revision all status.conditions asserts has been removed. (I think we can revist later, but the kuttl framework is picky and even errors if the number of elements here does not equal the actual number of elements.)